bin/vue-init

```diff
@@ -65,7 +65,7 @@ var name = inPlace ? path.relative('../', process.cwd()) : rawName
 var to = path.resolve(rawName || '.')
 var clone = program.clone || false
 
-var tmp = path.join(home, '.vue-templates', template.replace(/\//g, '-'))
+var tmp = path.join(home, '.vue-templates', template.replace(/[\/|:]/g, '-'))
```

Comment:
using `[]` will already do an _or_
I don't get your question, can you rephrase it, please?